packages/formation/sass/base/_b-breakpoints.scss

```diff
@@ -13,6 +13,16 @@
 // Based on the order of our imports, USWDS should override Foundation's declarations and everything should be fine.
 // However, USWDS defines its variables using the "!default" flag, so Foundation's take precedence anyway.
 // These next lines redeclare USWDS's variables without that flag, so that the override works as we would expect.
+
+$grid-columns-small: 1 !default;
```

Comment:
Reading the note above, did we get around to removing Foundation classes? I kept the `!default` flags in there, but wasn't sure.

Reply:
We've removed the visibility imports, I think, but the grid is a bigger project that we haven't gotten to yet.

Reply:
Is there any way you can confirm that these won't overwrite the values in vets-website?  

I assume because of the !default flag, it should be safe because it's supposed to just set the variable if it's not already set.

Reply:
Yeah, I believe that's how it works. It's not used in vets-website that I can see, so it should be safe. I think to test it specifically we'd need to make a class that used it and verify that the output was correct.